packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/media/VideoTrack.kt

```diff
@@ -24,4 +24,8 @@ open class VideoTrack(
   fun removeRenderer(renderer: VideoSink) {
     this.videoTrack.removeSink(renderer)
   }
+
+  fun setShouldReceive(shouldReceive: Boolean) {
```

Comment:
**suggestion:** 
I think that this function could be named shorter `shouldReceive`.